app/templates/_README.md

```diff
@@ -0,0 +1,89 @@
+# <%= humanName %>
+
+<%= props.description %>
+
+![Screenshot](screenshot.png)
+
+## Dependencies
+
+- [Grunt](http://gruntjs.com): task runner for pulling in assets,
+  linting and concatenating code, etc.
+- [Bower](http://bower.io): Package manager for front-end dependencies.
+- [Less](http://lesscss.org): CSS pre-processor.
+- [Capital Framework](https://cfpb.github.io/capital-framework/getting-started):
+  User interface pattern-library produced by the CFPB.
+
+**NOTE:** If you're new to Capital Framework, we encourage you to
+[start here](https://cfpb.github.io/capital-framework/getting-started).
+
+## Installation
+
+1. Install [Node.js](http://nodejs.org) however you'd like.
+2. Install [Grunt](http://gruntjs.com) and [Bower](http://bower.io):
+  ```bash
+  npm install -g grunt-cli bower
+  ```
+3. Install the Node and Bower dependencies from the project root:
+  ```bash
+  npm install
+  ```
+  **NOTE:** This will also run `bower install` and `grunt compile-cf` as post-install scripts
```

Comment:
Realized this file doesn't exist outside this PR so I updated the instructions here for the changes in #87.